src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1616,10 +1301,10 @@ public CitizenData getCitizen(final int citizenId)
     /**
      * Checks if the achievements are valid.
      */
-    private void checkAchievements()
+    public void checkAchievements()
     {
         // the colonies size
-        final int size = this.citizens.size();
+        final int size = this.citizenManager.getCitizens().size();
 
         //todo check those later again
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)